doc/source/user/numpy-for-matlab-users.rst

```diff
@@ -7,220 +7,124 @@ NumPy for MATLAB users
 Introduction
 ============
 
-MATLAB® and NumPy/SciPy have a lot in common. But there are many
-differences. NumPy and SciPy were created to do numerical and scientific
-computing in the most natural way with Python, not to be MATLAB clones.
-This page is intended to be a place to collect wisdom about the
-differences, mostly for the purpose of helping proficient MATLAB users
-become proficient NumPy and SciPy users.
+MATLAB® and NumPy have a lot in common, but NumPy was created to work with
+Python, not to be a MATLAB clone.  This guide will help MATLAB users get started
+with NumPy. 
 
 .. raw:: html
 
    <style>
    table.docutils td { border: solid 1px #ccc; }
    </style>
 
-Some Key Differences
+Some key differences
 ====================
 
 .. list-table::
 
```

Comment:
I think this will enable the formatting in the `.. raw` directive on line 14 (if I got the blank line and the number of spaces before the `:class:`)

```suggestion
.. list-table::
   :class: docutils

```